samples/cpp/benchmark_app/main.cpp

```diff
@@ -502,19 +516,21 @@ int main(int argc, char* argv[]) {
             // --------------------------------------------------------
             next_step();
             auto startTime = Time::now();
-            exeNetwork = ie.ImportNetwork(FLAGS_m, device_name, {});
+            compiledModel = core.compile_model(FLAGS_m, device_name, {});
```

Comment:
@vladimir-dudnik I believe this line is a root cause of the issue. You've replaced `Import` operation with `Compile`, but they are not equivalent.

@ilya-lavrenov @ilyachur What is a correct way to import pre-compiled blob in the new OV 2.0 API?

Reply:
@fzhar please change old API call ie.ImporModel() to new API call core.import_model()